src/main/java/io/supertokens/webserver/api/emailpassword/SignUpAPI.java

```diff
@@ -79,7 +79,9 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
 
         try {
             TenantIdentifierWithStorage tenant = this.getTenantIdentifierWithStorageFromRequest(req);
-            AuthRecipeUserInfo user = EmailPassword.signUp(tenant, super.main, normalisedEmail, password);
+            AuthRecipeUserInfo user = EmailPassword.signUp(
+                    tenant, super.main, normalisedEmail, password,
+                    getVersionFromRequest(req).greaterThanOrEqualTo(SemVer.v5_0));
```

Comment:
i don't think we need this extra boolean. The fake email is unique enough from a domain point of view. What do you think?

Reply:
yes, makes sense